config/crd/bases/cdap.cdap.io_cdapmasters.yaml

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
Were these removals intentional or auto-format?

Reply:
it's intentional removal